### DISCUSSION TITLE
Do the names in autostart matter?

### DISCUSSION THREAD

**Discussion by vdegenne** [WayfireWM/wayfire#2523] [Q&A] [answered]
Do the key names in autostart section matter?

https://github.com/WayfireWM/wayfire/blob/f66f23c0cf288f73cdb38ff992039fff2032e132/wayfire.ini#L142-L159

Or could I write for instance `whatever = mako` ?

I am asking because there is `autostart_wf_shell = true` which is a flag so are syntax like `notifications` also important to follow for wayfire integration?

**Comment by ammen99** [comment, marked as answer]
The only thing which matters is the autostart wf shell, the rest can be arbitrary (just not conflict with said option, and they shouldn't repeat)

**Reply by vdegenne** [reply]
Ok thanks. I am still in the settings right now, I don't know what happened I used to use gnome-center for turning dark mode on and off, it worked before but now turning dark mode on has no effect anymore... I tried using the portal in the default config file but nothing happens (even after reboot), what could be the problem?

**Reply by vdegenne** [reply]
Oh I needed the gtk portal nevermind.

**Reply by vdegenne** [reply]
@ammen99 One thing that I have noticed is that autostart commands that are commented in the config file will still show on the wayfire config UI, really confusing, are commented autostart entries running on boot too?

**Reply by soreau** [reply]
> @ammen99 One thing that I have noticed is that autostart commands that are commented in the config file will still show on the wayfire config UI, really confusing, are commented autostart entries running on boot too?

How are they commented? If the line starts with `#`, it should be ignored, unless you are modifying the config file while wcm is open.

**Reply by vdegenne** [reply]
I thought `;` was the way to comment 	( ͡° ᴥ ͡°)﻿
